### ISSUE TITLE
Adding pytest

### ISSUE THREAD

**Issue by jeffrey82221** [GrandMoff100/RegexFactory#11]
Hi @GrandMoff100 , 

Currently, I am trying to start building a regex generation project using your RegexFactory package. I would like to add some unittest  to your package so that I can be more confident in using the package. 

I wonder if I can fork this repo and contribute some unitests to it? 

Also, is there any thing I must notice when building the unittest?

**Comment by GrandMoff100** [comment]
Awesome!!!! I'd be honored if you contributed some tests to the project! As far what you should know this project is pretty simple. The examples folder is a good place to start. I'm almost certain that those examples should work and you can get the hang of the project fairly quickly. If they don't work I'll be happy to revitalize the project and collaborate with you on the tests.

**Comment by frank113** [comment]
Agreed. Regex operations are a constant annoyance to construct and this package is a simple way to add velocity when dealing with simple regular expressions. 

In terms of frameworks `pytest` is very powerful and extendable.

**Comment by GrandMoff100** [comment]
Yep `pytest` is my goto for building tests. I might build the tests myself pretty soon.
